packages/next-swc/crates/wasm/Cargo.toml

```diff
@@ -11,7 +11,6 @@ crate-type = ["cdylib"]
 anyhow = "1.0.42"
 console_error_panic_hook = "0.1.6"
 once_cell = "1.3.1"
-parking_lot_core = "=0.8.0"
```

Comment:
This dependency was causing conflicts and it didn't seem like it was being used so I removed it.

Reply:
You should not remove it, otherwise wasm build will fail

Reply:
Hmm... I looked at https://github.com/Amanieu/parking_lot/blob/80194730f2104fa5ca92fe17a619b57d0677ece7/core/src/parking_lot.rs and seems like it's fixed?